test-common/meson.build

```diff
@@ -19,11 +19,21 @@ libeos_updater_test_common_headers = [
   'utils.h',
 ]
 
+# ostree-trivial-httpd is in libexecdir/libostree, but we don't know
+# where libexecdir is. Try to build a path for typical places.
+ostree_trivial_httpd = find_program(
+    'ostree-trivial-httpd',
+    join_paths(libexecdir, 'libostree/ostree-trivial-httpd'),
+    '/usr/libexec/libostree/ostree-trivial-httpd',
+    '/usr/lib/libostree/ostree-trivial-httpd'
```

Comment:
Would be good to file an issue against OSTree and ask them to list their `libexecdir` in `ostree-1.pc`, then use that here and fall back to this lot if it’s not set.

Reply:
I was thinking about that but was a little concerned that `ostree-1.pc` really describes libostree and not the CLI or test programs. I.e., you could have `ostree-tests` installed without `libostree-dev` in Debian. Systemd has a similar issue where the pc files are split up and the one containing the unitdir setting is shipped in the main systemd package as opposed to the libsystemd-dev package. But I guess it would be better than nothing.